public/resources/core/js/controllers/product-offering.controller.js

```diff
@@ -197,6 +233,7 @@
             vm.data.productOfferingPrice.push(vm.pricePlan);
             vm.pricePlan = new Offering.PricePlan();
             vm.pricePlanEnabled = false;
+            vm.priceAlteration = 0;
```

Comment:
It is better for readability having an enumeration for this instead of just numbers